.github/workflows/ci.yml

```diff
@@ -190,10 +190,10 @@ jobs:
         with:
           save-if: ${{ github.ref == 'refs/heads/master' }}
 
+      - uses: r7kamura/rust-problem-matchers@v1
```

Comment:
Can you also set this one up for the following jobs please:

- rustfmt
- rustdoc links
- feature matrix
- cross
- test

Essentially, all jobs where we run `cargo`.

Ordering wise, I'd prefer to to be directly after the `rust-toolchain` action.

Reply:
Sure, I will do it.

Reply:
I added it for all the jobs that run cargo except from the ones that redirect the output to $GITHUB_OUTPUT